storage/aptosdb/src/state_store/mod.rs

```diff
@@ -367,9 +368,10 @@ impl StateStore {
         }
 
         // commit jellyfish merkle nodes
-        self.state_merkle_db.write_schemas(batch)?;
-
-        Ok(new_root_hash)
+        gauged_api("commit_jellyfish_merkle_nodes", || {
+            self.state_merkle_db.write_schemas(batch)?;
```

Comment:
Does this metric already provide what you want?
https://github.com/aptos-labs/aptos-core/blob/7bda1717bf137840705547b98f4e1639a3ec0038/storage/schemadb/src/lib.rs#L355

Reply:
I want to differentiate latency between state commit from state pruner. This metrics seem to be used by both.
<img width="1514" alt="Screen Shot 2022-07-08 at 2 45 31 PM" src="https://user-images.githubusercontent.com/107430656/178074513-20831cc4-10d3-4da2-85cd-7a02d4dcb8d2.png">

Reply:
I see. Can we use [OTHER_TIMERS_SECONDS](https://github.com/aptos-labs/aptos-core/blob/8d9073abe3439f8dfbebe499378e930d87619549/storage/aptosdb/src/metrics.rs#L102) instead since the one wrapped by gauged_api (API_LATENCY_SECONDS) is for AptosDB APIs.

Reply:
Done. Thanks!